### PR TITLE
feature/add_colleagues_table_on_front-end

### PR DESCRIPTION
**Pull request by VladyslavCherednychenko**
image
image
image

change functionality of colleagues request in UserInfoController
split ApiRequest file on three files one for each controller
add page to show direct colleagues of authorized user

### CODE REVIEW

File: Portal_Perfomance_Employes/Controllers/UserInfoController.cs [hunk 1]
```diff
@@ -99,7 +102,26 @@ public async Task<IActionResult> GetUserColleagues()
                 .FirstOrDefaultAsync(e => e.EmployeeId == id && e.IsActive);
             if (emp != null)
             {
-                var colleagues = await _context.TeamMembers.Where(c => c.TeamId == emp.TeamId && c.IsActive).ToListAsync();
+                var team = await _context.TeamMembers.Where(t => t.TeamId == emp.TeamId && t.IsActive).ToListAsync();
+
+                List<Employee> colleagues = new List<Employee>();
```
Comment (DmytroDorodnykhTeamInternational) [inline Portal_Perfomance_Employes/Controllers/UserInfoController.cs]: use var pls

File: Portal_Perfomance_Employes/Controllers/UserInfoController.cs [hunk 2]
```diff
@@ -99,7 +102,26 @@ public async Task<IActionResult> GetUserColleagues()
...
+                            Id = employee.Id,
+                            Username = employee.Username,
+                            EmailAddress = employee.EmailAddress,
+                            FirstName = employee.FirstName,
+                            LastName = employee.LastName,
+                            DateOfBirth = employee.DateOfBirth,
+                            Seniority = employee.Seniority,
+                            Role = employee.Role,
+                        });
+                    Console.WriteLine(await _context.Employees.Where(e => e.Id == colleague.EmployeeId).FirstOrDefaultAsync());
```
Comment (DmytroDorodnykhTeamInternational) [inline Portal_Perfomance_Employes/Controllers/UserInfoController.cs]: why? :)

File: react-portal-employees/src/components/menu/menuContainer.tsx [hunk 3]
```diff
@@ -15,7 +15,8 @@ import { isValid, isInvalid } from "../../redux/slice/authSlice";
 import { visitor, user, admin } from "../../redux/slice/roleSlice";
 import { useAppDispatch } from "../../redux/hooks";
 
-import { RefreshToken, GetRole } from "../../services/api/ApiRequests";
+import { RefreshToken } from "../../services/api/Requests/LoginControllersRequests";
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/menu/menuContainer.tsx:18]: why split???

File: react-portal-employees/src/components/signIn.tsx [hunk 4]
```diff
@@ -17,9 +17,8 @@ import Alert from "@mui/material/Alert";
 import Grid from "@mui/material/Grid";
 import Box from "@mui/material/Box";
 
-// import TokenValidate from "../services/tokenValidate";
-import { AuthRequest } from "../services/api/ApiRequests";
-import { GetRole } from "../services/api/ApiRequests";
+import { AuthRequest } from "../services/api/Requests/LoginControllersRequests";
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/signIn.tsx:20]: there and everywhere

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]

File: Portal_Perfomance_Employes/Controllers/UserInfoController.cs (hunk 1 above)
Comment (VladyslavCherednychenko) [reply Portal_Perfomance_Employes/Controllers/UserInfoController.cs]: okay

File: Portal_Perfomance_Employes/Controllers/UserInfoController.cs (hunk 2 above)
Comment (VladyslavCherednychenko) [reply Portal_Perfomance_Employes/Controllers/UserInfoController.cs]: sorry, forgot to delete

**Review by dariia-kachalova — APPROVED** [review]

**Review by Ygoraphael — APPROVED** [review]

**Review by dangoth — APPROVED** [review]

**Pull request merged**